doc/src/performance_tuning.rst

```diff
@@ -0,0 +1,151 @@
+.. highlight:: none
+
+******************
+Performance Tuning
+******************
+
+OpenMP Backend
```

Comment:
It might also be worth stressing that this section is about performance *tuning*, and that if your performance is a long way from what might be considered reasonable then these points might be so useful. A non-exhaustive list of common issues with performance might be a good idea, but equally, this can be a bit of a pandora's box.

Reply:
Done